src/main/java/Main.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+
+public class Main {
```

Comment:
I suggest removing this class since there is no important logic. We have tests to check if logic is written correctly.